spyder/plugins/variableexplorer/widgets/tests/test_dataframeeditor.py

```diff
@@ -544,7 +549,7 @@ def test_dataframeeditor_edit_complex(qtbot, monkeypatch):
     """
     Test that editing complex dtypes is handled gracefully in df editor.
 
-    Integration regression test for issue #6115 .
+    Integration regression test for spyder-ide/spyder#6115 .
```

Comment:
Although not related to the actual change, this texts sounds a bit odd.

Maybe 

```
Integration and regression test for spyder-ide/spyder#6115
```

or
```
Integration test for regression spyder-ide/spyder#6115
```